mitodl/open-discussions
# Move widget/group creation for fields to signal

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes #3687

#### What's this PR do?
Moves code for creating a field channel widget and group/role from the serializer to a signal.  Makes channel image attributes blankable to make editing easier from django admin.

#### How should this be manually tested?
- Create a new `FieldChannel` in django admin.  Verify that it has an associated `WidgetList`
- Create a new `FieldChannel` via the API at `/api/v0/fields` (post this data: `{"name": "newfield1", "title": "New Field 1"}`.  Verify that it has an associated `WidgetList` and there is a `Group` for it (named `field_<name>_moderators`)
